system/assets/js/admin/presidecore/preside.richeditor.js

```diff
@@ -18,6 +18,7 @@ PresideRichEditor = ( function( $ ){
 		  , enterMode             = $elementToReplace.data( "enterMode" )
 		  , autoParagraph         = $elementToReplace.data( "autoParagraph" ) !== undefined ? $elementToReplace.data( "autoParagraph" ) : cfrequest.ckeditorAutoParagraph
 		  , defaultConfigs        = cfrequest.ckeditorDefaultConfigs || {}
+		  , customDefaultConfigs  = $elementToReplace.data( "custom-default-configs" ) || {}
```

Comment:
Hi @john5onCheng 

Although `.data( "custom-default-configs" )` will work, the jQuery convention is that dashes in the HTML data attribute get converted to camelCase in jQuery.

So preferable would be `.data( "customDefaultConfigs" )`, similar to the lines above.